CHANGELOG.md

```diff
@@ -35,6 +35,13 @@ version directory, and then changes are introduced.
 ### Added
 - Disable API etcd data encryption experimental feature.
 
+### Changed
+- Updated calico to 2.6.5.
```

Comment:
I just noticed it. Verbs should be in imperative form. I.e. `Update`, `Remove`, etc.

Reply:
Also when this is merged version bundles in kvm-operator and aws-operator should be updated.